UI/JustAssembly/ViewModels/ShellViewModel.cs

```diff
@@ -86,6 +96,48 @@ public void OpenNewSessionCommandExecuted()
                 this.OnLoadCommandExecuted(newSessionViewModel.SelectedSession);
             }
         }
+            else
+            {
+                Configuration.Analytics.TrackFeature("DiffSessionType.CommandLine." + newSessionViewModel.SelectedSession.SelectedItemType.ToString());
```

Comment:
The analytics library expect maximum of 2 dot-separated segments, i.e. only 1 dot is permitted. My suggestion for here is this - leave the current analytics call to work always, but make new one with feature "CommandLineParameters.NewFileSessionFiles" only if there are passed arguments.

Reply:
I changed the tracking (**CommandLineParameters.NewFileSessionFiles**) and added a new one for when the "invalid args" dialog is shown. (**CommandLineParameters.InvalidArgsDialogShown**)